libcontainer/nsenter/nsexec.c

```diff
@@ -1053,6 +1083,14 @@ void nsexec(void)
 					bail("failed to sync with parent: SYNC_MOUNTSOURCES_ACK: got %u", s);
 			}
 
+			/*
+			 * Update timens offsets
+			 * set boottime and monotonic offsets
+			 */
+			write_log(DEBUG, "set timens offsets %s", config.boottime);
```

Comment:
You're outputting this debug information twice (both here and in update_timens). You can remove this one.